test/jasmine/tests/hover_label_test.js

```diff
@@ -2663,14 +2663,14 @@ describe('hover on traces with (x|y)period positioning', function() {
         .then(function() {
             assertHoverLabelContent({
                 name: 'scatter',
-                nums: '(Jan 2001, 1)'
+                nums: '(Jul 2001, 1)'
```

Comment:
So these tests were actually locking down the incorrect behavior previously... This is why in general I think such tests should be self-contained rather than relying on a separate mock: you can read the test carefully and if the dimensions are nice and simple (ie you set round numbers for the size, margins, & axis ranges) you can manually verify that it's the desired behavior.